client/my-sites/plan-features/style.scss

```diff
@@ -95,6 +95,33 @@ $plan-features-header-banner-height: 20px;
 	margin: 0;
 	font-size: 28px;
 	line-height: 1.5;
+	color: $gray-dark;
+}
+
+.plan-features__price.is-original {
+	color: $gray;
+}
+
+.plan-features__price.is-discounted {
+	color: $alert-green;
+}
+
+.plan-features__price.is-discounted,
+.plan-features__price.is-original {
+	position: relative;
+	align-items: stretch;
+	margin-right: 8px;
+}
+
+.plan-features__price.is-original:before {
+	position: absolute;
+	content: "";
+	left: 0;
+	top: 50%;
+	right: 0;
+	border-top: 2px solid $orange-fire;
+	transform: rotate(-16deg);
```

Comment:
Only `JPY` goes into that range, and their smallest unit is 1 yen, so we won't see any decimals. I think the current CSS should be fine.